### PR TITLE
Add CHANGELOG entry for 7.0 release

### PR DESCRIPTION
**Pull request by byucesoy**
This PR adds CHANGELOG entry for our 7.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,70 @@
...
-* Replaces pg_dist_shard_placement metadata table with pg_dist_placement
+* Adds support for PostgreSQL 10
+
+* Drops support for PostgreSQL 9.5
+
+* Adds support for multi-row `INSERT`
+
+* Adds support for router `UPDATE` and `DELETE` queries with subqueries
+
+* Adds infrastructure to distributed deadlock detection
```
Comment (metdos) [inline CHANGELOG.md]: to -> for

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,70 @@
...
+
+* Fixes a bug causing failure during `COPY` on tables with dropped columns
+
+* Fixes a bug causing failure during `DROP EXTENSION`
+
+* Fixes a bug preventing executing `VACUUM` and `INSERT` concurrently
+
+* Fixes a bug in prepared `INSERT` statements containing an implicit cast
+
+* Fixes several issues related with statement cancellations and connections
```
Comment (metdos) [inline CHANGELOG.md]: with -> to

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,70 @@
...
+
+* Fixes several 2PC related issues
+
+* Removes an unnecessary dependency causing warning messages in pg_dump
+
+* Replaces `pg_dist_shard_placement` metadata table with `pg_dist_placement`
```
Comment (metdos) [inline CHANGELOG.md]: I think we still have `pg_dist_shard_placement` as least a view.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,70 @@
...
+
+* Drops support for PostgreSQL 9.5
+
+* Adds support for multi-row `INSERT`
+
+* Adds support for router `UPDATE` and `DELETE` queries with subqueries
+
+* Adds infrastructure to distributed deadlock detection
+
+* Adds maintenance daemon for distributed deadlock detection and 2PC recovery
```
Comment (metdos) [inline CHANGELOG.md]: I think we don't do 2PC recovery automatically yet. If that is the case, I think we can drop this line.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,70 @@
...
+
+* Adds support for subquery pushdown on reference tables
+
+* Adds shard pruning support for `IN` and `ANY`
+
+* Adds support for `UPDATE` and `DELETE` commands that prune down to 0 shard
+
+* Enhances transaction support by relaxing some transaction restrictions
+
+* Extends transaction support to more operations
```
Comment (metdos) [inline CHANGELOG.md]: These two lines seem to do the same thing. Shall we drop one?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,70 @@
...
+
+* Fixes a bug causing crash if distributed table has no shards
+
+* Fixes a bug causing crash when removing inactive node
+
+* Fixes a bug causing failure during `COPY` on tables with dropped columns
+
+* Fixes a bug causing failure during `DROP EXTENSION`
```
Comment (marcocitus) [inline CHANGELOG.md]: Is this the issue introduced by distributed deadlock detection or another one?

Comment (byucesoy) [reply CHANGELOG.md]: No, this was existing bug before deadlock detection.

**Comment by metdos** [comment]
Looks good 🚢

**Pull request merged**